internal/service/kubernetes/node_group.go

```diff
@@ -144,7 +144,17 @@ func ResourceNodeGroup() *schema.Resource {
 				Default:     true,
 				ForceNew:    true,
 			},
-			"storage_encryption": storageEncryptionSchema("Storage encryption strategy for the nodes in this group.", true),
+			"storage_encryption": {
+				Description: "Storage encryption strategy for the nodes in this group.",
+				Type:        schema.TypeString,
+				Computed:    true,
+				Optional:    true,
+				ForceNew:    true,
+				ValidateDiagFunc: validation.ToDiagFunc(validation.StringInSlice([]string{
+					string(upcloud.StorageEncryptionDataAtReset),
```

Comment:
Should this be _data at rest_ instead of _data at reset_? Would have to be fixed in the Go SDK, though 🤔

Reply:
Ah, good catch :smile: :+1: I'll create SDK PR to fix that.